ada-project-docs/wave_01.md

```diff
@@ -25,27 +25,22 @@ It's crucial for all APIs to be able to handle errors. For every required endpoi
 
 If something goes wrong, your API should return:
 - an appropriate [HTTP status code](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status)
-- a list of errors
+- For POST and PUT requests, responses with 4XX response codes should also return an array of error messages to explain what is wrong with the request.
 
 For this project, the list of errors should be formatted like this:
 
 ```json
-{
-  "errors": {
-    "available_inventory": [
-      "can't be blank",
-      "is not a number"
-    ]
-  }
-}
+// for a video endpoint
+[
+   "title must be provided and it must be a string",
+   "total_inventory must be provided and it must be a number"
+]
 
-// ...or...
+// ...or...for a rental endpoint
 
```

Comment:
rather than the earlier change, I'd suggest removing this blank line so the comment hugs the following data.
```suggestion
```